parsec/core/logged_core.py

```diff
@@ -329,6 +335,18 @@ async def start_greeting_device(self, token: UUID) -> DeviceGreetInProgress1Ctx:
         initial_ctx = DeviceGreetInitialCtx(cmds=self._backend_conn.cmds, token=token)
         return await initial_ctx.do_wait_peer()
 
+    async def get_organization_status(self) -> OrganizationStatus:
```

Comment:
I find it a bit confusing to have both "organization_status" and "organization_stats". What's the difference between the two?
If they trully are separate, should'nt we rename "organization_status" to something else just in the sake of avoiding confusion?

Reply:
I used the same name than the backend.
The organization_stats API return the organization's metrics (users, storage, etc.) while the organization_status API return data about the state of the organization. (is bootstraped, deprecation date, is outsider enabled, etc.)
I already thought to rename this API because there is an admin command with the same name. We discussed about it with @touilleMan and the conclusion was to keep the name because the HandshakeTypes were differents.

Reply:
Given they already is `organization_status`/`organization_stats` in the administration API, we cannot change the naming without breaking compatibility (or following a different naming between the administration and the authenticated API, but I find it also confusing...)

Reply:
Sounds like configuration to me, maybe we should call it `organization_config`?